flow/core/kernel/vehicle/traci.py

```diff
@@ -293,10 +293,8 @@ def _add_departed(self, veh_id, veh_type):
         self.__vehicles[veh_id]["type"] = veh_type
 
         # specify energy model
-        energy_model = \
-            self.type_parameters[veh_type]["energy_model"]
-        self.__vehicles[veh_id]["energy_model"] = \
-            energy_model[0](veh_id, **energy_model[1])
+        self.__vehicles[veh_id]["energy_model"] = self.type_parameters[
+            veh_type]["energy_model"]()
```

Comment:
is this guaranteed to not be none?

Reply:
ya, it defaults to some energy model (it's never None)